config/printer-sunlu-t3-2022.cfg

```diff
@@ -0,0 +1,211 @@
+# Printer
+##################################################################
+
+[mcu]
+#obtain your MCU id using ls /dev/serial/by-path/*
+serial: dev/serial/by-id/usb-Klipper_stm32f103xe_833E31383534300530343833-if00
+
+[printer]
+kinematics: cartesian
+max_velocity: 300
+max_accel: 3000
+max_z_velocity: 5
+max_z_accel: 100
+
+[static_digital_output usb_pullup_enable]
+pins: !PA14
+
+[bltouch]
+sensor_pin: PC14
+control_pin: PA1
+x_offset: -28.45
+y_offset: 4
+z_offset: 1.915  #recheck you own
+pin_up_touch_mode_reports_triggered: FALSE #needed bc of the bltouch clone used by sunlu
+
+[safe_z_home]
+home_xy_position: 115,115
+speed: 75
+z_hop: 10
+z_hop_speed: 5
+
+[bed_mesh]
+speed: 120
+horizontal_move_z: 5
+mesh_min: 10, 10
+mesh_max: 190, 220
+probe_count: 5,5
+fade_start: 1
+fade_end: 10
+
+[bed_screws]
+#for BED_SCREWS_ADJUST
+screw1: 31,38 #X,Y Position
+screw1_name: Front Left
+screw2: 201,38 #X,Y Position
+screw2_name: Front Right
+screw3: 201,204 #X,Y Position
+screw3_name: Rear Right
+screw4: 31,204 #X,Y Position
+screw4_name: Rear Left
+
+
+##IMPORTANT. Add if using the filament sensor add CLEAR_PAUSE to your slicer's start gcode or to your print start macro.##
+##The act of loading and unloading filament will trigger a paused state##
+[filament_motion_sensor Filament_Sensor]
+detection_length: 7.0
+#   The minimum length of filament pulled through the sensor to trigger
+#   a state change on the switch_pin
+#   Default is 7 mm.
+extruder: extruder
+#   The name of the extruder section this sensor is associated with.
+#   This parameter must be provided.
+switch_pin: !PC15
+pause_on_runout: FALSE
+runout_gcode: PAUSE
+#insert_gcode:
+#event_delay:
+#pause_delay:
+#   See the "filament_switch_sensor" section for a description of the
+#   above parameters.
+
+#########################################################
+# Motion Axis
+#########################################################
+
+[stepper_x]
+step_pin: PB13
+dir_pin: !PB12
+enable_pin: !PB14
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PC0
+position_endstop: 0
+position_max: 235
+homing_speed: 50
+
+[stepper_y]
+step_pin: PB10
+dir_pin: !PB2
+enable_pin: !PB11
+microsteps: 16
+rotation_distance: 40
+endstop_pin: !PC1
+position_endstop: 0
+position_max: 235
+homing_speed: 50
+
+[stepper_z]
+step_pin: PB0
+dir_pin: PC5
+enable_pin: !PB1
+microsteps: 16
+rotation_distance: 4
+position_max: 250
+endstop_pin: probe:z_virtual_endstop
+
+###################################################
+# Heaters
+###################################################
+
+[extruder]
+step_pin: PB3
+dir_pin: !PB4
+enable_pin: !PD2
+microsteps: 16
+rotation_distance: 23.18840579710145  #verify your own
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PC8
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PA0
+control: pid
+pid_Kp: 19.479 #calibrate your own PID
+pid_Ki: 1.073
+pid_Kd: 88.385
+min_extrude_temp: 175
```

Comment:
Please do not override safety settings, min_extruder_temp and max_extrude_only_distance should be removed.

Reply:
Sorry about that. I had those in my own config for use with filament load/unload macros and forgot to remove them when i removed those macros. Removed now.